### PR TITLE
Added Patch call for StorSimple Manager. Removed SupportPackageReques…

### PR DESCRIPTION
**Pull request by manaas-microsoft**
…t and ListAccessKey Operations.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by manaas-microsoft** [comment]
Hi, This is 5th PR (of the set of 6 scheduled PRs). In this, we have added the Manager Patch call, which was pending from earlier PRs. Also, we have removed two operations, namely, ListAccessKey and PublishSupportPackage(since, client SDK work PR is not yet raised, I hope it is fine).

 Also, the examples for each operations is a work in progress, and once all the test recordings for SDK for net is done, we will extracting the examples and adding here.

Please do let me know, if anything else is required.

**Comment by manaas-microsoft** [comment]
The other minor fixes are:
(a) For the global parameter, deviceNameParameter, earlier we had missed adding the property that it is should be part of method and found it while working on the SDK test recordings. 
(b) CloudAppliance --> CloudAppliances was required for fixing OperationIdNounConflictingModelNames. For the controller name, it had somehow got missed earlier. 
(c) x-ms-pageable --> This has also got missed for one of the listMetrics operations, but on regeneration of swagger, it came correct now.

File: arm-storsimple8000series/2017-06-01/swagger/storsimple.json [hunk 1]
```diff
@@ -243,6 +243,51 @@
           }
         },
         "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "Managers"
+        ],
+        "description": "Updates the StorSimple Manager.",
+        "operationId": "Managers_UpdateManager",
```
Comment (jhendrixMSFT) [inline arm-storsimple8000series/2017-06-01/swagger/storsimple.json]: This should be Managers_Update.

Comment (manaas-microsoft) [inline arm-storsimple8000series/2017-06-01/swagger/storsimple.json]: Sure. Will make that change and upload now.

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**